packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -460,6 +460,9 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       createEsqlRuleType: `${SECURITY_SOLUTION_DOCS}rules-ui-create.html#create-esql-rule`,
       entityAnalytics: {
         riskScorePrerequisites: `${SECURITY_SOLUTION_DOCS}ers-requirements.html`,
+        hostRiskScore: 'host-risk-score.html',
```

Comment:
I ran the code locally, and from what I can tell these actually set the href to these values directly, instead of having the domain, etc.

I've attached an example screenshot after I inspected it in Chrome. Note the `href` value specifically.

<img width="1727" alt="Screenshot 2023-11-29 at 1 33 04 PM" src="https://github.com/elastic/kibana/assets/147995946/fa6c9bdc-7dad-4385-82f9-d793019e53d3">

Is this a limitation of running locally? Else, I assume we need to make the following change (for each):
```suggestion
        hostRiskScore: '${SECURITY_SOLUTION_DOCS}host-risk-score.html',
```

Reply:
🤦

Reply:
Resolved by https://github.com/elastic/kibana/pull/172172/commits/4b07190bfe33ea79228773b1ba71314238137ec0